src/GameSaves/OriginalGameSaveToFileWriter.cs

```diff
@@ -777,7 +785,7 @@ public void WriteSaveGame(GameSave gameSave)
                     bw.Write((byte)0);
                 }
 
-                // TODO: AI opponents
+                // TODO: AI opponents (shoulnd't be - 1?)
                 bw.Write((ushort)(gameSave.Players.Count - 2));
```

Comment:
Perhaps a better way to write this is something like:
`bw.Write((ushort)(gameSave.Player.Count(x => !(x.Civilization is Barbarian) && !x.IsHuman)));`